plasmapy/particles/tests/test_particle_collections.py

```diff
@@ -44,6 +44,8 @@ def various_particles():
             CustomParticle(),
             CustomParticle(mass=7 * u.kg),
             CustomParticle(charge=11 * u.C),
+            5 * u.kg,
+            5 * u.C,
```

Comment:
So these two will create, respectively, a charge-less 5kg chonkyboi of a particle and a massless one with a massive 5 C charge, both custom ones, right?

Reply:
Yep!